image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -153,6 +159,12 @@ spec:
           mountPath: /run/secrets/stackrox.io/db-password
         - name: central-external-db-volume
           mountPath: /etc/ext-db
+        {{- if ._rox.central.notifierSecretsEncryption }}
```

Comment:
I'm not sure if we need both if's here or if only `{{- if ._rox.central.notifierSecretsEncryption.enabled }}` is enough.

Did you have problems with CI, your local tests when we didn't have both?

Reply:
I was getting many CI failures when _rox.central.notifierSecretsEncryption doesn't exist.